src/archivist/filter/exports.js

```diff
@@ -0,0 +1,3 @@
+import filter from './index.js';
```

Comment:
```suggestion
export { default as filter } from './index.js';
```

Reply:
This will export

```
{ filter: filterFunction}
```

And not export the default function

This `import otaFilter from 'open-terms-archive/filter';` would end with `otaFilter` being `undefined`

So this is not what I want to achieve.

See https://jakearchibald.com/2021/export-default-thing-vs-thing-as-default/
and https://stackoverflow.com/questions/39999282/re-export-default-in-es-6-modules

From the second link

- `export { default } from './App'` does not work on my environment
- `export default from "./App.js"` is currently under proposal

So I will stick with what I did unless you point me to a working solution which I would be glad to find